### PR TITLE
Added --ignore-namespace to emmocheck 

### PR DESCRIPTION
**Pull request by francescalb**
It is now possible to ignore the a given namespace by:

emmocheck --ignore-namespace=http://emmo.info/emmo/emmo-inferred# emmo-inferred

However, this is not the same as ignoring submodules so does not work correctly yet. When  you load an ontology, the base_iri is the same for the whole ontology...

### CODE REVIEW

File: emmo/emmocheck.py [hunk 1]
```diff
@@ -281,6 +283,10 @@ def main():
     parser.add_argument(
         '--url-from-catalog', '-u', action='store_true',
         help=('Get url from catalog file'))
+    parser.add_argument(
+        '--ignore-namespace', '-in', action='append', default=[],
```
Comment (jesper-friis) [inline emmo/emmocheck.py:287]: Unusual with 2-letter short option. Normally -in is a shortcut for "-i -n" if "-i" doesn't take an argument or "-i n" if "-i" takes an argument.

**Review by jesper-friis — COMMENTED** [review]
See inline comment

File: emmo/emmocheck.py (hunk 1 above)
Comment (francescalb) [reply emmo/emmocheck.py]: thank you, this is fixed

**Comment by francescalb** [comment]
Works for skipping namespace check for 'emmocheck --check-imported --ignore-namespace=emmo-inferred emmo-inferred'. However namespace check for emmo does not go into the check at all because when onto = get_ontolog('emmo').load() all of the following are empty: onto.classes(),                         onto.object_properties(), onto.data_properties(), onto.individuals(), onto.annotation_properties(). Is this correct behaviour?

**Review by jesper-friis — APPROVED** [review]
Great Francesca. Thanks

**Pull request merged**